file/interface.go

```diff
@@ -0,0 +1,10 @@
+package file
```

Comment:
Do we use this interface in this package? Might be moved back in elrond-go

Reply:
no. removed.